lib/Graph/Graph.cpp

```diff
@@ -1478,9 +1478,9 @@ Function::createRowwiseQuantizedSparseLengthsSum(llvm::StringRef name,
 
 FusedRowwiseQuantizedSparseLengthsWeightedSumNode *
 Function::createFusedRowwiseQuantizedSparseLengthsWeightedSum(
-    llvm::StringRef name, Constant *data, NodeValue weights, NodeValue indices,
+    llvm::StringRef name, NodeValue data, NodeValue weights, NodeValue indices,
```

Comment:
can you explain this change?

Reply:
lowerNode(... Node* n ...) receives a pointer to Node as an argument and passes a const SpecificNode& to lowerSpecificNode(..), which later calls createSpesific(NodeValue nv).

In this case "Specific" is "FRQSLWS" which is also used by ModelLoader that passes it a Constant* which is why the existing signature.
Now, when I want to use it also from lowerSpecificNode(..) I cannot cast/convert from a const SpecificNode& to NodeValue. However I can implicitly convert from Constant* to NodeValue (since Constant is derived from Node and NodeValue wraps Node*).  
That looked like a reasonable thing to do...